covid19/weather.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+"""Script to extract and transform weather data using the DarkSky API"""
+from typing import Optional
+
+import pandas as pd
+import requests
+
+from covid19 import utils
+from covid19.config import API_KEY
+
+
+def get_weather_data(api_token: str, lat: float, lon: float, date: str) -> dict:
```

Comment:
https://docs.python.org/3/library/typing.html look there, i think you need to `from typing import Dict`

Reply:
Updated imports to now be `from typing import Optional, Dict`